staticfiles/templates/contributors/home.html

```diff
@@ -208,7 +208,14 @@ <h1 class="page-header">
               <img class="thumbnail img-responsive" src="/static/images/contributors/milstein.png" />
               <span>Milson Munakami</span> 
             </a>
-          </div>		  
+          </div>
+          
+          <div class="col-sm-2 col-xs-4 thumb">
+            <a class="contributor-thumb" href="/contributors/suvansh">
+              <img class="thumbnail img-responsive" src="/static/images/contributors/suvansh.jpg" />
```

Comment:
Should I upload it to the pull request itself?
On May 28, 2015 8:40 PM, "neilthemathguy" <notifications@github.com> wrote:

> In staticfiles/templates/contributors/home.html
> <https://github.com/crowdresearch/crowdsource-platform/pull/180#discussion_r31297014>
> :
>
> > @@ -208,7 +208,14 @@ <h1 class="page-header">
> >                <img class="thumbnail img-responsive" src="/static/images/contributors/milstein.png" />
> >                <span>Milson Munakami</span>
> >              </a>
> > -          </div>		
> > +          </div>
> > +
> > +          <div class="col-sm-2 col-xs-4 thumb">
> > +            <a class="contributor-thumb" href="/contributors/suvansh">
> > +              <img class="thumbnail img-responsive" src="/static/images/contributors/suvansh.jpg" />
>
> suvansh.jpg is missing. Could you upload it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crowdresearch/crowdsource-platform/pull/180/files#r31297014>
> .
>